src/docs/examples/lists/SmallList.tsx

```diff
@@ -14,30 +14,30 @@ export default function VerticalList() {
       <Preview>Centered Paragraph</Preview>
       <Body style={main}>
         <Container style={container}>
-          <ListRoot size="xs">
+          <ListRoot>
             <ListItem>
-              <ListItemTitle>
+              <ListItemTitle size="xs">
                 Using content you’ve shared publicly
               </ListItemTitle>
-              <ListItemContent>
+              <ListItemContent size="xs">
                 For example, to promote a Google app, we might quote a review
                 you wrote. Or to promote Google Play, we might show a screenshot
                 of the app you offer in the Play Store.
               </ListItemContent>
             </ListItem>
             <ListItem>
-              <ListItemTitle>
+              <ListItemTitle size="xs">
                 Developing new technologies and services
               </ListItemTitle>
-              <ListItemContent>
+              <ListItemContent size="xs">
                 For Google consistent with these terms
               </ListItemContent>
             </ListItem>
             <ListItem>
-              <ListItemTitle>
+              <ListItemTitle size="xs">
                 Operating and improving the services
               </ListItemTitle>
-              <ListItemContent>
+              <ListItemContent size="xs">
```

Comment:
Definetely aware, and I think it would be ideal to offer common styling from list element (less cases in our examples than I expected but would expect more in real world emails).
Good idea with CSS, as I dont have training with email testing I would suggest exploring that in consequent feature ticket as I am not sure how quickly we can develop/test that. But if well supported and possible to combine with list item level overrides, then it could be best solution.

Reply:
IMO we can merge this now and worry about the CSS styling in another issue/PR